tests/unit/test_metrics.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+import bentoml
+import pytest
+import sys
+
+@pytest.mark.parametrize("metrics_type", filter(lambda x: isinstance(x, bentoml.metrics.BaseMetric), dir(bentoml.metrics)))
+def test_metrics_initialization(metrics_type: t.Type[bentoml.metrics.BaseMetric]):
+    m = metrics_type(name="test_metrics", documentation="test")
+    assert "prometheus_client" not in sys.modules
+
+# TODO: 
+# - Test that prometheus_client gets initiated after metric gets called
+# - Test that metric is found in serve instance after it's initiated
```

Comment:
remove todo

Reply:
done; Are there any guide lines on when I can/cannot use TODO? Or should I just avoid using TODO all together?

Reply:
no need to add todo when you already address all of it.